quarkdown-html/src/test/resources/rendering/block/heading.html

```
@@ -80,4 +80,10 @@ <h3 id="foo-bar">
 
 <h4 id="foo-bar">
     Foo bar
-</h4>
\ No newline at end of file
+</h4>
+
+---
+
+<h2 id="foo-bar" style="color: rgba(255, 0, 0, 1.0); font-variant: small-caps;">
```

HTMLHint (1.9.2)

[error] The id value [ foo-bar ] must be unique.

(id-unique)